src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -52,6 +58,18 @@
 public final class RsWithCookie extends RsWrap {
 
     /**
+     * Cookie value validation regexp.
+     * @checkstyle LineLengthCheck (2 lines)
+     */
+    private static final Pattern CVALUE_PTRN = Pattern.compile("[\\x21\\x23-\\x2B\\x2D-\\x3A\\x3C-\\x5B\\x5D-\\x7E]*|\"[\\x21\\x23-\\x2B\\x2D-\\x3A\\x3C-\\x5B\\x5D-\\x7E]*\"");
```

Comment:
@yegor256  regexp too long here
It doesn't allow to remove checkstyle suppression